usecases/stable/documentationcamera/recording.documentationcamera.realtime/odcockpit/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Dockerfile - Dockerfile to run OpenDLV software.
+# Copyright (C) 2016 Christian Berger
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Date: 2016-09-09
+
+FROM seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete:latest
```

Comment:
In the stable folder, we only allow use cases that point to a fixed version that we know is working.

Everything else must reside in latest to avoid misunderstandings.

I am aware that we are currently changing many things related to camera/velodyne but we will settle this soon with the next testing round I suppose.